main/autocode/src/org/ejml/GenerateJavaCode32.java

```diff
@@ -50,6 +50,10 @@ public GenerateJavaCode32() {
 
         prefix64.add("DGrow");
         prefix32.add("FGrow");
+        prefix64.add("DoubleUnary");
```

Comment:
Thoughts on changing this to DUnary and DBinary? It's more in keeping with the existing naming scheme. It is a little bit less readable though.

Reply:
Right, I chose Double for readability. But it is better to stick with the existing naming scheme to avoid confusions

Reply:
Just to give you some background, the names use to be longer a while ago but things got so cumbersome when the number of data types increased that these abbreviations started to be used.